lib/Elastica/QueryBuilder/DSL/Query.php

```diff
@@ -196,7 +195,7 @@ public function field()
      */
     public function filtered(AbstractQuery $query = null, AbstractFilter $filter = null)
     {
-        return new Filtered($query, $filter);
+        throw new DeprecatedException('Use BoolQuery instead. Filtered query is deprecated since ES 2.0.0-beta1 and this class will be removed in further Elastica releases.');
```

Comment:
I'm not sure, but if Filtered query just deprecated now, but still can be used, than better here just trigger deprecated error, instead of throw Exception. Also I think here nice to add deprecated info to phpdoc.